integration/test_auditlogs.py

```diff
@@ -1,13 +1,19 @@
 from datetime import datetime
 from datetime import timedelta
 
+import pytest
 from integration import run_command
 
-BASE_COMMAND = "code42 auditlogs search -b"
+BASE_COMMAND = "code42 audit-logs search -b"
```

Comment:
Failing command change and below change is just a refactor to use `pytest.mark.parametrize`, using it helps us to view the executed command during test execution.